app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/Utils.java

```diff
@@ -16,34 +16,11 @@
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.OutputStream;
-import java.util.Locale;
 
 public class Utils {
 
     public static final int MY_PERMISSIONS_REQUEST_CAMERA = 1;
     public static final int MY_PERMISSIONS_REQUEST_STORAGE= 2;
-    private static final String OPEN_FOOD_FACTS_WORLD_URL = "http://world.openfoodfacts.org/";
-    private static final String HTTP_FR_OPENFOODFACTS_ORG = "http://fr.openfoodfacts.org/";
-
-    public static String getUriByCurrentLanguage() {
-        String url;
-        if (Locale.getDefault().getLanguage().contains("fr")){
-            url = HTTP_FR_OPENFOODFACTS_ORG;
-        } else {
-            url = OPEN_FOOD_FACTS_WORLD_URL;
-        }
-        return url;
-    }
-
-    public static String getUriProductByCurrentLanguage() {
-        String url;
-        if (Locale.getDefault().getLanguage().contains("fr")){
-            url = HTTP_FR_OPENFOODFACTS_ORG + "produit/";
-        } else {
-            url = OPEN_FOOD_FACTS_WORLD_URL + "product/";
-        }
```

Comment:
For the web version, we have put the "slugs": produit, contributeur… in separate strings.